plugins/modules/os_default_template.py

```diff
@@ -36,6 +36,24 @@
       - name of the template kind
     required: true
     type: str
+    choices:
```

Comment:
this is not a doc fragment, because the two modules use different names for that property :roll_eyes:

Reply:
also, the lists differ, the one module accepts "snippet", the other does not.